index.html

```diff
@@ -169,13 +169,13 @@ <h3 class="languages-h3">Skills</h3>
     <section class="touch">
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
     </section>
-    <article class="form">
-        <input type="text" placeholder="Full name">
-        <input type="email" placeholder="Email address">
-        <textarea name="" id="" cols="30" rows="10">Enter text here</textarea>
-    </article>
+    <form class="form" action="https://formspree.io/f/xpznodbr" method="POST">
+        <input class="common" type="text" placeholder="Full name" required maxlength="30">
+        <input class="common" type="email" placeholder="Email address" required>
```

Comment:
- [ ] I will suggest you remove the height you added to your form inputs and then increase the padding to match the Figma design you chose 👍🏽 . Your input fields are looking too wide, kindly fix it

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/200674192-9ef4e39d-e9b2-4ba6-b70d-b296fcb49a88.png)

FIgma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/200674493-50b4e4c1-faa8-40aa-998c-a9c2ad72361b.png)

Reply:
Fixed the input height.